### DISCUSSION TITLE
可以提供一个配置主题的组件吗

### DISCUSSION THREAD

**Discussion by QiYuOr2** [fect-org/fect#166] [Ideas]
类似于`<ThemeProvider :theme=""></ThemeProvider>`这种，可以通过这个组件实现对于一些组件内置样式的修改，这样要实现黑暗模式或者是自定义类似于颜色的样式会轻松许多

**Comment by nonzzz** [comment]
这个是可以纳入下个版本的迭代考虑的对此你有什么好的建议吗？目前看来采用css variable 进行替换会有些许繁琐？@QiYuOr2

**Comment by nonzzz** [comment]
1.1.0版本开始会提供一个内置的`hooks`,`use-theme` 展示效果和文档一样。未来会在1.2.0考虑纳入`theme-provider`组件进行用户自定义`css-variable`

**Comment by nonzzz** [comment]
参照issue #168 
我们将在下个版本提供css variable 和 构建时替换的方案让用户自定义`css variable`
